src/machine/behavior/is/mod.rs

```diff
@@ -19,15 +19,263 @@ pub fn get(_opcode: u8) -> Semantic {
 }
 
 type Semantic = fn(&mut State, u8, u8, u8);
-/*
-pub const SEMANTICS: [Semantic; 256] = [
-                        // symbol   hex  dec
-
-    interrupt::trap,    // TRAP     #00    0
-    float::fcmp,        // FCMP     #01    1
-
-    /* TODO */          // PUSHGOI  #BF  191
 
-    other::swym         // SWYM     #FF  255
+pub const SEMANTICS: [Semantic; 256] = [
+    interrupt::trap,    
```

Comment:
To be honest: I have not seen that you wrote that within the pull request's description.

For the moment: I will approve these changes if we add an issue for that. If I have time in the end, I will write all these comments.

For the one who writes the comments: There is a programming technique called meta-programming. You write a program that writes programs. I use this technique with bash, the pipe (|), sed, egrep, cat, ...